### DISCUSSION TITLE
Only one channel on a zone?

### DISCUSSION THREAD

**Discussion by realaod** [micro-nova/AmpliPi#125] [Q&A] [answered]
Hello,

I have 2 zones that only have a single channel on them. I didn't design it. Anyhow, would it be okay to just have one channel connected to a zone or is it recommended to run both channels to the single speaker? I know some hardware it is fine to just run a single channel on a zone and other hardware it is recommended to output both channels to the one speaker.

Thanks!

**Comment by linknum23** [comment, marked as answer]
Not a problem at all. We have a single speaker hooked up to one of our development units right now and it sounds great. The only issue I can see is wasting an amplified output. Lemme check with our hardware engineer just to be sure though. @gorski123 is there anything I am missing here?

Just a heads up, in case you ever want to upgrade, they make single unit stereo speakers for smaller rooms. Here is an example: https://www.crutchfield.com/S-p5Y5Uxb9h9U/p_107RC6S/Polk-Audio-RC6s.html. Jason has a speaker like this in each of his bathrooms and the laundry room.

**Comment by gorski123** [comment]
There should be no problem only running a single channel on a zone. As Lincoln mentioned we have done it in the past. I would however verify that your single speaker is not actually a stereo speaker. I have a few in my house that are actually stereo speakers. To verify you can pull the speaker and check the connections - a stereo speaker will have two sets (4 wires) of speaker connections on the back of it - one for the left channel and one for the right.

**Comment by realaod** [comment]
Thank you both for the quick answers! You are right, they might actually be stereo. I haven't looked since they are obviously covered with a grill. I'll take a look before the Amplipi arrives. I'm so stoked about this thing!
